pkgs/development/tools/sourcetrail/default.nix

```diff
@@ -120,7 +120,7 @@ in stdenv.mkDerivation rec {
 
     # Sourcetrail attempts to copy clang headers from the LLVM store path
     substituteInPlace CMakeLists.txt \
-      --replace "\''${LLVM_BINARY_DIR}" '${llvmPackages.clang-unwrapped}'
+      --replace "\''${LLVM_BINARY_DIR}" '${llvmPackages.clang-unwrapped.lib}'
```

Comment:
More future proof?

```suggestion
      --replace "\''${LLVM_BINARY_DIR}" '${lib.getLib llvmPackages.clang-unwrapped}'
```

Reply:
I would have done that to start with if I knew it worked.